ocaml/tests/test_repository_helpers.ml

```diff
@@ -1163,13 +1090,18 @@ module EvalGuidanceForOneUpdate = Generic.MakeStateless (struct
                       id= "UPDATE-0000"
                     ; summary= "summary"
                     ; description= "description"
-                    ; rec_guidance= Some Guidance.RebootHost
+                    ; rec_guidance= Some RebootHost
```

Comment:
Do we need the old fields `rec_guidance` and `abs_guidance` for the new update? As we need new client(new version of XC) to talk to xapi.

Reply:
Yes. It's fine to get rid of them from internal data structure.